gradle.properties

```diff
@@ -20,4 +20,6 @@ kotlin.code.style=official
 # Enables namespacing of each library's R class so that its R class includes only the
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
-android.nonTransitiveRClass=true
\ No newline at end of file
+android.nonTransitiveRClass=true
+# Enable R8 full mode.
+android.enableR8.fullMode=true
```

Comment:
R8 Full Mode is [enabled by default in AGP 8.0.0+ ](https://developer.android.com/build/releases/past-releases/agp-8-0-0-release-notes#default-changes)